scripts/scil_compute_centroid.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import save_tractogram
+from dipy.segment.clustering import QuickBundles
+from dipy.segment.metric import ResampleFeature
+from dipy.segment.metric import AveragePointwiseEuclideanMetric
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_reference_arg)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description='Compute bundle centroid',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    p.add_argument('in_bundle',
+                   help='Fiber bundle file.')
+    p.add_argument('out_centroid',
+                   help='Output centroid streamline filename.')
+
+    p.add_argument('--distance_thres',
```

Comment:
If the goal is to obtain a single centroid and nothing else. Why is this even a parameters, let's just use 1000000 ?
The goal is to average ALL streamlines into a single centroid. anything too low will crash, it doesn't support negative. Simply using infinity could work 100% of the time, no ?

Reply:
Or to save a ton of line of code. Manually resample to NBR_POINTS and then simply use np.average on the appropriate axis.

Reply:
I agree